chart/values.yaml

```diff
@@ -184,6 +184,14 @@ resources: {}
   #  cpu: 100m
   #  memory: 128Mi
 
+## Autoscaling parameters
+autoscaling:
+  enabled: false
+  # minReplicas: 1
```

Comment:
We might want to do some cleanup so that we don't cause confusion between `minReplicas` with hpa and the default `replicaCount` configuration.

Reply:
Sure - do you have any preference to the solution? AFAIK, the `replicaCount` defines how many pods are started on initial deployment, while `minReplicas` is simply bounding how small the HPA can scale the pod count to - so they're not mutually exclusive options.

Reply:
Hmm.. This is true. My initial thought was just that it would be confusing to have a specific number set by `replicaCount` and then have it scale below that because of `minReplicas`.

Reply:
Are you happy to accept this as-is?